### PR TITLE
Update install instructions

### PR DESCRIPTION
**Pull request by emodric**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -56,11 +56,13 @@ Example: In the case of stock eZ Platform 1.11 and higher that would specificall
 ```
 
 
-### Enable EzPublishLegacyBundle
+### Enable EzPublishLegacyBundle and optionally EzSystemsEzPlatformXmlTextFieldTypeBundle
```
Comment (andrerom) [inline INSTALL.md]: it's not optional anymore.

Comment (andrerom) [reply INSTALL.md]: or, well people can indeed skip to register it, so maybe ignore this

File: INSTALL.md [hunk 2]
```diff
@@ -101,10 +103,8 @@ _Tip:_
 
 `ezsystems/legacy-bridge` contains the libraries previously included in `ezsystems/ezpublish-kernel` in version 5.x.
```
Comment (andrerom) [inline INSTALL.md]: `contains a newer version of the libraries previously included in`

File: INSTALL.md [hunk 3]
```diff
@@ -56,11 +56,13 @@ Example: In the case of stock eZ Platform 1.11 and higher that would specificall
 ```
 
 
-### Enable EzPublishLegacyBundle
+### Enable EzPublishLegacyBundle and optionally EzSystemsEzPlatformXmlTextFieldTypeBundle
 Edit `app/AppKernel.php`, and add `new eZ\Bundle\EzPublishLegacyBundle\EzPublishLegacyBundle( $this ),`
 at the end of  the `$bundles` array. Pay close attention to the `$this` argument, LegacyBundle is a bit 
 spoiled and has high expectations from its collaborators ;)
 
+If you need support for legacy `ezxmltext` field type in eZ Publish kernel, enable `EzSystems\EzPlatformXmlTextFieldTypeBundle\EzSystemsEzPlatformXmlTextFieldTypeBundle` bundle.
```
Comment (andrerom) [inline INSTALL.md]: maybe something like:
`If you still plan to have content in XmlText _(as opposed to migrating to RichText straight away)_, then you'll also need to enable ...`

Comment (andrerom) [reply INSTALL.md]: the updated version is +1 as well

**Comment by emodric** [comment]
@andrerom Nitpicks fixed.

File: INSTALL.md [hunk 4]
```diff
@@ -56,11 +56,15 @@ Example: In the case of stock eZ Platform 1.11 and higher that would specificall
...
 
-### Enable EzPublishLegacyBundle
+### Enable EzPublishLegacyBundle and EzSystemsEzPlatformXmlTextFieldTypeBundle
 Edit `app/AppKernel.php`, and add `new eZ\Bundle\EzPublishLegacyBundle\EzPublishLegacyBundle( $this ),`
 at the end of  the `$bundles` array. Pay close attention to the `$this` argument, LegacyBundle is a bit 
 spoiled and has high expectations from its collaborators ;)
 
+If you still plan to have content in XmlText _(as opposed to migrating to RichText straight away)_, then
+you'll also need to enable `EzSystems\EzPlatformXmlTextFieldTypeBundle\EzSystemsEzPlatformXmlTextFieldTypeBundle`
+bundle.
```
Comment (andrerom) [inline INSTALL.md]: and maybe `... bundle, which gets installed together with legacy-bridge.`

Comment (emodric) [reply INSTALL.md]: Done.

**Pull request merged**